server/views/components/sessionTemplateVisitStats.njk

```diff
@@ -0,0 +1,37 @@
+{%- from "moj/components/banner/macro.njk" import mojBanner -%}
+
+{% macro sessionTemplateVisitStats(visitStats) %}
```

Comment:
Moved this visit stats banner component to its own macro to avoid duplication of it in both the view / edit templates.